packages/core/src/button/Button.tsx

```diff
@@ -93,43 +79,29 @@ export const Button = forwardRef(function Button<
     };
   }, [active, keyIsDown]);
 
-  const handleKeyUp = (event: KeyboardEvent<inferElementType<T>>) => {
+  const handleKeyUp = (event: KeyboardEvent<HTMLButtonElement>) => {
```

Comment:
If this active logic is the same between components could we create a useActiveStyle or something hook?

Reply:
We could also do a useButton hook and include focusWhenDisabled